rero_ils/modules/documents/views.py

```diff
@@ -323,3 +325,44 @@ def item_status_text(item, format='medium', locale='en'):
             if item.status == ItemStatus.IN_TRANSIT:
                 text += ' ({0})'.format(_(ItemStatus.IN_TRANSIT))
     return text
+
+
+@blueprint.app_template_filter()
+def identifiedby_format(identifiedby):
+    """Format identifiedby for template."""
+    output = []
+    for identifier in identifiedby:
+        status = identifier.get('status')
+        id_type = identifier.get('type')
+        if status not in ('invalid', 'cancelled', 'invalid or cancelled') and \
+                id_type != 'bf:Local':
+            if id_type.startswith('bf:'):
```

Comment:
If the goal is to remove the prefix, perhaps it would be better to be more generic and remove any prefix by removing any char until the first ':'